docs/kb/rules/ellipsis-metavariables.md

```diff
@@ -0,0 +1,40 @@
+---
+description: Ellipsis metavariables can help with matching multiple word tokens.
+tags:
+  - Rules
+  - Semgrep Code
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Matching multiple tokens with ellipsis metavariables
+
+## What are ellipsis metavariables?
+
+Using ellipsis (`...`) to match a sequence of items is one of the most common constructs in Semgrep rules, and ellipses can even be used with metavariables ($VAR) to increase matching scope!
```

Comment:
Are items the same as tokens?

Reply:
I think probably if we get tokens defined correctly above, yes. :)